composer.json

```diff
@@ -1,15 +1,34 @@
 {
-    "name": "wpcomvip/s3-media-sync",
-    "type": "wordpress-plugin",
-    "require": {
-        "php": "^8.1",
-        "composer/installers": "~1.0 || ^2.0",
-        "aws/aws-sdk-php": "~3.288.1"
-    },
-    "config": {
-        "allow-plugins": {
-            "composer/installers": true,
-            "dealerdirect/phpcodesniffer-composer-installer": true
-        }
-    }
+	"name": "wpcomvip/s3-media-sync",
+	"type": "wordpress-plugin",
+	"require": {
+		"php": "^8.1",
+		"composer/installers": "~1.0 || ^2.0",
+		"aws/aws-sdk-php": "~3.288.1"
+	},
+	"config": {
+		"allow-plugins": {
+			"composer/installers": true,
+			"dealerdirect/phpcodesniffer-composer-installer": true
+		}
+	},
+	"require-dev": {
+		"phpunit/phpunit": "^9.0",
+		"yoast/phpunit-polyfills": "^1.0",
```

Comment:
```suggestion
```

Not needed, as wp-test-utils already [includes phpunit-polyfills](https://github.com/Yoast/PHPUnit-Polyfills/blob/134921bfca9b02d8f374c48381451da1d98402f9/composer.json#L30) as its own dependency.